features/languages.feature

```diff
@@ -1,76 +1,75 @@
 @Languages
 Feature: Languages
-@AWS
-Scenario: Ensure you can change languages in CommCare
-	Then I install the ccz app at "languages.ccz"
-	Then I login with username "user_with_no_data" and password "123"
 
-	#Check that default language deployed
-	Then I press start
-	Then I select module "Basic Form Test"
-	And I select form "Languages"
+  Scenario: Ensure you can change languages in CommCare
+    Then I install the ccz app at "languages.ccz"
+    Then I login with username "user_with_no_data" and password "123"
 
-	Then I see the text "Enter a name:"
-	Then I go back to the home screen
+   #Check that default language deployed
+    Then I press start
+    Then I select module "Basic Form Test"
+    And I select form "Languages"
 
-	#Change the language from home screen menu
-	Then I select "Change Language" menu item
-	Then I see the text "Hindi"
-	Then I see the text "English"
+    Then I see the text "Enter a name:"
+    Then I go back to the home screen
 
-	#Persists on rotation
-	Then I rotate to landscape
-	Then I see the text "Hindi"
-	Then I see the text "English"
-	
-	Then I rotate to portrait
-	Then I see the text "Hindi"
-	Then I see the text "English"
-	Then I press "Hindi"
+   #Change the language from home screen menu
+    Then I select "Change Language" menu item
+    Then I see the text "Hindi"
+    Then I see the text "English"
 
-	#Confirm the change 
-	Then I press start
-	Then I select module "Basic Form Tests"
-	And I select form "HIN: Languages"
+   #Persists on rotation
+    Then I rotate to landscape
+    Then I see the text "Hindi"
+    Then I see the text "English"
 
-	Then I see the text "HIN: Enter a name:"
-	Then I go back to the home screen
+    Then I rotate to portrait
+    Then I see the text "Hindi"
+    Then I see the text "English"
+    Then I press "Hindi"
 
-	Then I press start
-	Then I select module "Basic Form Tests"
-	And I select form "HIN: Languages"
+   #Confirm the change
+    Then I press start
+    Then I select module "Basic Form Tests"
+    And I select form "HIN: Languages"
 
-	Then I select "Change Language" menu item
-	Then I see the text "en"
-	Then I see the text "hin"

```

Comment:
It's hard to tell unless you look at this with `?w=1`, but these got changed from `Then I see the text "en"` to `Then I see the text "English"` and `Then I see the text "hin"` to `Then I see the text "Hindi"`, respectively. Was that intentional?